src/Akka.Streams.Kafka/Settings/ConsumerSettings.cs

```diff
@@ -360,4 +415,16 @@ public sealed class ConsumerSettings<TKey, TValue>
                 .Build();
         }
     }
+
+    internal sealed class RebalanceListener<TKey, TValue>
```

Comment:
What's this for? I thought we already had a rebalance listener implementation.

Reply:
This is an internal class used only in tests. Its job is to pass the internal callback handlers set up inside the consumer actor to the mock consumer to emulate a real callbacks during runtime.